src/main/cli/cli.c

```diff
@@ -1252,25 +1253,51 @@ static int cliParseSerialMode(const char *tok)
     return mode;
 }
 
+static int cliParseSerialOptions(const char *tok)
+{
+    int options = 0;
+
+    if (strcasestr(tok, "bidir")) {
+        options |= SERIAL_BIDIR;
+    }
+    if (strcasestr(tok, "bidir_pp")) {
```

Comment:
bidir_pp will always trigger bidir branch too ...

Reply:
this is wanted as just SERIAL_BIDIR_PP dont workes alone. so you can have just SERIAL_BIDIR or SERIAL_BIDIDR | SERIAL_BIDDIR_PP

Reply:
OK, but this is terrible design practise - this is far from obvious and it _will_ break in future.
At least comment this behavior and use `options |= SERIAL_BIDIR | SERIAL_BIDIR_PP;` in this branch

Reply:
done :)